core/domain/services/admin/privacy/erasure/EraseAnswers.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace EventEspresso\core\domain\services\admin\privacy\erasure;
```

Comment:
erasure?
like the 80s synth pop band?

Reply:
never heard of them before... listening... yes, this is named after that band, and their tambourine.